ELClient/ELClient.cpp

```diff
@@ -81,6 +83,11 @@ ELClientPacket* ELClient::protoCompletedCb(void) {
     }
     return NULL;
   } else {
+    if( callbackPacketHandler != NULL )
+    {
+      if( callbackPacketHandler(packet) )
```

Comment:
This is not a response. It's more like a topic to subscribe.
I didn't use this because of its name.